packages/web-components/src/components/ic-select/ic-select.tsx

```diff
@@ -54,193 +54,162 @@ export class Select {
   private menu: HTMLIcMenuElement;
```

Comment:
Some of these aren't in alphabetical order

Same with the state variables below